src/main/java/pl/piekoszek/gorskimatches/equation/EquationGenerator.java

```diff
@@ -7,9 +7,10 @@
 @Component
 public class EquationGenerator {
```

Comment:
It's no longer `EquationGenerator` it's quiz generator

Reply:
or create class `QuizzGenerator` as separate component and make it use `EquationGenerator`, that would be much better